packages/contracts-bedrock/contracts/L1/L1CrossDomainMessenger.sol

```diff
@@ -17,7 +17,8 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, Semver {
     /**
      * @notice Address of the OptimismPortal.
      */
-    OptimismPortal public immutable portal;
+    // solhint-disable-next-line var-name-mixedcase
+    OptimismPortal public immutable PORTAL;
```

Comment:
Only some places need legacy getters. This was a new variable as of Bedrock, so no legacy getter required.